neatnet/tests/test_simplify.py

```diff
@@ -108,3 +108,29 @@ def test_simplify_network_full_fua(aoi, tol, known_length):
             observed.drop(columns=["_status", "geometry"]),
         )
         pytest.geom_test(known, observed, tolerance=tol, aoi=aoi)
+
+
+@pytest.mark.wuhan
+def test_simplify_network_wuhan(aoi="wuhan_8989", tol=0.3, known_length=4_702_861):
```

Comment:
My thought here would be to have a single helper that runs the guts of this function + `test_simplify_network_full_fua()` and is called within each. That could also be adapted for `test_simplify_network_small()`. But that can wait until another PR for sure (if we decide to do).